### ISSUE TITLE
FASTA rename syntax error 

### ISSUE THREAD

**Issue by Oelsakha** [rotheconrad/00_in-situ_GeneCoverage#3]
hi all 
when I run this code gives me syntax error 
the original code:
_python 01b_Fasta_rename_sequences.py -i metagenome_file_name.fastq -p metagenomeID_
My code:
_python xsss204.fasta -i xsss204.fasta -p xsss204_

I have the last python version Python 3.9.12

and also this code give my error 
python scriptname.py -h
thank you

**Comment by rotheconrad** [comment]
Can you copy and paste your full command you entered and the full error message? Thanks you

**Comment by Oelsakha** [comment]
python xsss204.fasta rename xsss204.py -i xsss204.fasta -p xsss204
  File "xsss204.fasta", line 1
    >CP0866789.2 Staphylococcus aureus strain xsss204 chromosome, complete genome
    ^
SyntaxError: invalid syntax

**Comment by rotheconrad** [comment]
I see. The script name needs to be included after you call python.

python 01b_Fasta_rename_sequences.py -i xsss204.fasta -p xsss204

**Comment by Oelsakha** [comment]
I tried this before and give me this error do you think that I need to download any thing?
python: can't open file '01b_Fasta_rename_sequences.py': [Errno 2] No such file or directory

**Comment by rotheconrad** [comment]
Yes. You need to download the files from this GitHub repository or you need to give the full file path to where you have the .py file.

**Comment by Oelsakha** [comment]
It works. Thank you so much
